.github/workflows/maven-verify.yml

```diff
@@ -28,7 +28,9 @@ jobs:
     with:
       # maven-args: "-Dinvoker.parallelThreads=2" cannot do this as this generate some concurrent download issues
       jdk-distribution-matrix: '[ "temurin", "zulu", "microsoft", "adopt-openj9" ]'
-      ff-jdk: "20"
-      jdk-matrix: '[ "8", "11", "17", "20" ]'
-      matrix-exclude: '[{ "jdk": "8", "distribution": "microsoft"},{ "jdk": "20", "distribution": "microsoft"},{ "jdk": "20", "distribution": "adopt-openj9"},]'
-      maven-matrix: '[ "3.6.3", "3.8.8", "3.9.5" ]'  # Maven versions matrix for verify builds
+      matrix-exclude: '[
```

Comment:
the goal was to have a maximum of jdk to be tested as compiling can be highly dependant of the used jdk.
so with this we are really reducing the coverage.
could we keep testing on 11 maybe?

Reply:
done

Reply:
@slawekjaranowski Thanks! I was thinking of m-javadoc-p if we have similar "cleanup" :)